src/NCalc.Sync/Visitors/EvaluationVisitor.cs

```diff
@@ -239,7 +234,7 @@ public void Visit(Identifier identifier)
     protected int CompareUsingMostPreciseType(object? a, object? b)
     {
         return TypeHelper.CompareUsingMostPreciseType(a,
-            b, new(Context.CultureInfo,
+            b, new TypeHelper.ComparasionOptions(Context.CultureInfo,
```

Comment:
>TypeHelper.ComparasionOptions

A possible typo? Should be ComparisonOptions IMO.

Reply:
It's because it's a scoped type inside TypeHelper. I accidentally ctrl+. here, but I will change soon